CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+ * [`fixed`]    Fix typo in header include-guard now.
```

Comment:
Is it relevant for the users? Maybe because previously one could get conflicts if multiple headers used the `GIT_VERSION_H`.

Reply:
most probably not, but if we release now we'll have an empty changelog

Reply:
Let's keep it then.